src/testdata/daemon-config.txt

```diff
@@ -0,0 +1,18 @@
+# This is a valid test configuration.  Some of the spacing is wonky in order to test
```

Comment:
"daemon-config.txt" is a bad name for this.  "daemon-stdio-config.txt" might be better.  Plus a comment to that effect.